backend/coins/eth/etherscan/etherscan.go

```diff
@@ -116,6 +116,7 @@ type jsonTransaction struct {
 	BlockNumber jsonBigInt     `json:"blockNumber"`
 	GasUsed     jsonBigInt     `json:"gasUsed"`
 	GasPrice    jsonBigInt     `json:"gasPrice"`
+	Nonce       jsonBigInt     `json:"nonce"`
```

Comment:
Can't really find anything on the wire format in https://etherscan.io/apis#transactions but I guess you know better.

Reply:
Apart from that, what if the `nonce` is nil or in a different format. Wouldn't that basically break the app in a way users won't be able to do anything with ETH? ...since it's `jsonBigInt`'s unmarshaler always assumes it must be valid.

For example, `txlist` action in `EtherScan.Transactions`. Does it guarantee that the nonce value is always valid?

Reply:
yeah every tx has a nonce, i think that should be no problem.